src/components/TextStack/TextStack.module.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../styles/colors';
+
+.Container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 0.25rem;
+
+    font-size: 0.75rem;
+
+    color: grey;
```

Comment:
is this using the color variable? might need to be `$grey-lighter-2`

Reply:
Will keep until `ThemeEngine` is merged